k8s/namespaces/ccd/ccd-api-gateway-web/perftest-redirect.yaml

```
@@ -0,0 +1,21 @@
+# this is to apply permanent redirect on www-ccd.perftest.platform.hmcts.net
+---
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  annotations:
+    kubernetes.io/ingress.class: traefik
+    traefik.ingress.kubernetes.io/redirect-permanent: "true"
+    traefik.ingress.kubernetes.io/redirect-regex: ^https://www-ccd.perftest.platform.hmcts.net/(.*)
```

```suggestion
    traefik.ingress.kubernetes.io/redirect-regex: ^https://www-ccd.perftest.platform.hmcts.net(.*)
```
With the `/`, will it work for users who just try going to `https://www-ccd.perftest.platform.hmcts.net`?